pages/upgrade-guide/4.0.md

```diff
@@ -0,0 +1,186 @@
+<script>{
+	"title": "jQuery Core 4.0 Upgrade Guide",
+	"toc": true
+}</script>
+
+## Overview
```

Comment:
We want that for the RC, before the final, so before the final gets released, I'd add a note here at the top that jQuery 4.0 is in the RC phase and some details may still change before the stable release.